dev/validate_bom__deploy.py

```diff
@@ -1230,17 +1278,17 @@ def init_argument_parser(parser, defaults):
   # pylint: disable=line-too-long
   add_parser_argument(
       parser, 'halyard_install_script', defaults,
-      'https://raw.githubusercontent.com/spinnaker/halyard/master/install/nightly/InstallHalyard.sh',
+      'https://raw.githubusercontent.com/spinnaker/halyard/master/install/debian/InstallHalyard.sh',
```

Comment:
This script defaults to "stable", since it is the replacement for the stable install script. Maybe the `halyard_version` parameter above should do the same now?

Reply:
Not sure I follow. The halyard_version is left empty by default and only passed in if set. So by default it wont specify anything, which will pick up the default in the script, which is 'stable'.
The intent here is to use the default, whatever it may happen to be, unless explicitly specified.

Reply:
I'm just pointing out this is a change in behavior -- the default (using the script that was deleted in this diff) was 'nightly'. Now it's 'stable'.

Reply:
ok. that's actually good. in practice we were injecting stable by default in the processes anyway.